### ISSUE TITLE
installation issue

### ISSUE THREAD

**Issue by anasred** [cpriego/valet-linux#47]
I run this command
```
sudo composer global require cpriego/valet-linux
```

And this is what i got:
```
Changed current directory to /home/anas/.composer
Using version ^2.0 for cpriego/valet-linux
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - cpriego/valet-linux v2.0.1 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.10 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.11 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.2 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.3 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.4 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.5 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.6 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.7 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.8 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - cpriego/valet-linux v2.0.9 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - nategood/httpful 0.2.9 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.8 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.7 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.6 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.5 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.3 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.20 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.2 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.19 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.17 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.16 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.13 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.11 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.10 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.1 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.0 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - Installation request for cpriego/valet-linux ^2.0 -> satisfiable by cpriego/valet-linux[v2.0.1, v2.0.10, v2.0.11, v2.0.2, v2.0.3, v2.0.4, v2.0.5, v2.0.6, v2.0.7, v2.0.8, v2.0.9].

  To enable extensions, verify that they are enabled in those .ini files:
    - /etc/php/7.0/cli/php.ini
    - /etc/php/7.0/cli/conf.d/10-mysqlnd.ini
    - /etc/php/7.0/cli/conf.d/10-opcache.ini
    - /etc/php/7.0/cli/conf.d/10-pdo.ini
    - /etc/php/7.0/cli/conf.d/15-xml.ini
    - /etc/php/7.0/cli/conf.d/20-calendar.ini
    - /etc/php/7.0/cli/conf.d/20-ctype.ini
    - /etc/php/7.0/cli/conf.d/20-dom.ini
    - /etc/php/7.0/cli/conf.d/20-exif.ini
    - /etc/php/7.0/cli/conf.d/20-fileinfo.ini
    - /etc/php/7.0/cli/conf.d/20-ftp.ini
    - /etc/php/7.0/cli/conf.d/20-gd.ini
    - /etc/php/7.0/cli/conf.d/20-gettext.ini
    - /etc/php/7.0/cli/conf.d/20-iconv.ini
    - /etc/php/7.0/cli/conf.d/20-intl.ini
    - /etc/php/7.0/cli/conf.d/20-json.ini
    - /etc/php/7.0/cli/conf.d/20-mbstring.ini
    - /etc/php/7.0/cli/conf.d/20-mcrypt.ini
    - /etc/php/7.0/cli/conf.d/20-mysqli.ini
    - /etc/php/7.0/cli/conf.d/20-pdo_mysql.ini
    - /etc/php/7.0/cli/conf.d/20-phar.ini
    - /etc/php/7.0/cli/conf.d/20-posix.ini
    - /etc/php/7.0/cli/conf.d/20-readline.ini
    - /etc/php/7.0/cli/conf.d/20-shmop.ini
    - /etc/php/7.0/cli/conf.d/20-simplexml.ini
    - /etc/php/7.0/cli/conf.d/20-sockets.ini
    - /etc/php/7.0/cli/conf.d/20-sysvmsg.ini
    - /etc/php/7.0/cli/conf.d/20-sysvsem.ini
    - /etc/php/7.0/cli/conf.d/20-sysvshm.ini
    - /etc/php/7.0/cli/conf.d/20-tokenizer.ini
    - /etc/php/7.0/cli/conf.d/20-wddx.ini
    - /etc/php/7.0/cli/conf.d/20-xmlreader.ini
    - /etc/php/7.0/cli/conf.d/20-xmlwriter.ini
    - /etc/php/7.0/cli/conf.d/20-xsl.ini
  You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.

Installation failed, reverting ./composer.json to its original content.

```

**Comment by anasred** [comment]
Oops my bad.
php curl extension were missin.

So run
```
sudo apt-get install php7.0-curl
```
and proplem solved!!

**Comment by rhchristian** [comment]
thx for the solution.

**Comment by victorlt** [comment]
:D The perfect solution.

**Comment by petruskubito** [comment]
not work to me

**Comment by rockyru** [comment]
sudo apt-get install php7.0-curl is not working for me. any more solution? thanks!

**Comment by anasred** [comment]
@rockyru what's the error message?

**Comment by petruskubito** [comment]
root@bash:~/code/multiauth# composer global require laravel/valet
Changed current directory to /home/petrus/.composer
Using version ^2.0 for laravel/valet
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/valet v2.0.0 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - laravel/valet v2.0.1 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - laravel/valet v2.0.3 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - laravel/valet v2.0.4 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - laravel/valet v2.0.5 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - laravel/valet v2.0.6 requires nategood/httpful ~0.2 -> satisfiable by nategood/httpful[0.2.0, 0.2.1, 0.2.10, 0.2.11, 0.2.13, 0.2.16, 0.2.17, 0.2.19, 0.2.2, 0.2.20, 0.2.3, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9].
    - nategood/httpful 0.2.9 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.8 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.7 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.6 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.5 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.3 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.20 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.2 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.19 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.17 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.16 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.13 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.11 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.10 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.1 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - nategood/httpful 0.2.0 requires ext-curl * -> the requested PHP extension curl is missing from your system.
    - Installation request for laravel/valet ^2.0 -> satisfiable by laravel/valet[v2.0.0, v2.0.1, v2.0.3, v2.0.4, v2.0.5, v2.0.6].

  To enable extensions, verify that they are enabled in those .ini files:
    - /etc/php/7.0/cli/php.ini
    - /etc/php/7.0/cli/conf.d/10-mysqlnd.ini
    - /etc/php/7.0/cli/conf.d/10-opcache.ini
    - /etc/php/7.0/cli/conf.d/10-pdo.ini
    - /etc/php/7.0/cli/conf.d/15-xml.ini
    - /etc/php/7.0/cli/conf.d/20-calendar.ini
    - /etc/php/7.0/cli/conf.d/20-ctype.ini
    - /etc/php/7.0/cli/conf.d/20-dom.ini
    - /etc/php/7.0/cli/conf.d/20-exif.ini
    - /etc/php/7.0/cli/conf.d/20-fileinfo.ini
    - /etc/php/7.0/cli/conf.d/20-ftp.ini
    - /etc/php/7.0/cli/conf.d/20-gd.ini
    - /etc/php/7.0/cli/conf.d/20-gettext.ini
    - /etc/php/7.0/cli/conf.d/20-iconv.ini
    - /etc/php/7.0/cli/conf.d/20-json.ini
    - /etc/php/7.0/cli/conf.d/20-mbstring.ini
    - /etc/php/7.0/cli/conf.d/20-mysqli.ini
    - /etc/php/7.0/cli/conf.d/20-pdo_mysql.ini
    - /etc/php/7.0/cli/conf.d/20-phar.ini
    - /etc/php/7.0/cli/conf.d/20-posix.ini
    - /etc/php/7.0/cli/conf.d/20-readline.ini
    - /etc/php/7.0/cli/conf.d/20-shmop.ini
    - /etc/php/7.0/cli/conf.d/20-simplexml.ini
    - /etc/php/7.0/cli/conf.d/20-sockets.ini
    - /etc/php/7.0/cli/conf.d/20-sysvmsg.ini
    - /etc/php/7.0/cli/conf.d/20-sysvsem.ini
    - /etc/php/7.0/cli/conf.d/20-sysvshm.ini
    - /etc/php/7.0/cli/conf.d/20-tokenizer.ini
    - /etc/php/7.0/cli/conf.d/20-wddx.ini
    - /etc/php/7.0/cli/conf.d/20-xmlreader.ini
    - /etc/php/7.0/cli/conf.d/20-xmlwriter.ini
    - /etc/php/7.0/cli/conf.d/20-xsl.ini
  You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.

Installation failed, reverting ./composer.json to its original content.

**Comment by petruskubito** [comment]
can you help me ?

**Comment by rockyru** [comment]
@anasred here's the error. I'm using php7.1. I've installed all the necessary dependencies.
![image](https://user-images.githubusercontent.com/13403945/34280825-7fe874ee-e6f5-11e7-9518-e267ff1c2e4a.png)

**Comment by TheNodi** [comment]
@petruskubito The error says you don't have php-curl installed. Are you sure it's available? Can you see it in your php info?

@rockyru I think you have some globally installed packages that are conflicting with Valet. Do you have some Symfony 4 packages?

**Comment by petruskubito** [comment]
oke thanks this works for me :+1:

**Comment by rockyru** [comment]
@TheNodi It's working, resolved the issue by using cgr.

**Comment by TheNodi** [comment]
@rockyru Great, unfortunately global composer packages don't have a great isolation and sometimes conflict. CGR is a perfect solution. 

Symfony 4 packages will be supported in the next version of Valet, so you might be able to not need CGR soon-ish.

**Comment by rockyru** [comment]
@TheNodi haha! That's a great news! Thanks, @TheNodi.

**Comment by victorlt** [comment]
Very Nice, with the version php7.1 working very nice this solution.

**Comment by Nixonkimeli** [comment]
Awesome!!!working for php 7.2 Thanks

**Comment by gilbenpratama** [comment]
I'm a windows user, do u have any idea bro?

**Comment by emretuna** [comment]
`sudo apt-get install php7.*-curl`
use if you still get the error.

**Comment by felexkemboi** [comment]
@emretuna `sudo apt-get install php7.*-curl` worked for me perfectly.Thank you

**Comment by daniyum21** [comment]
still doesn't seem to work for me:

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install cpriego/valet-linux v2.1.10
    - Conclusion: don't install cpriego/valet-linux v2.1.9
    - Conclusion: don't install cpriego/valet-linux v2.1.8
    - Conclusion: don't install cpriego/valet-linux v2.1.7
    - Conclusion: don't install cpriego/valet-linux v2.1.6
    - Conclusion: don't install cpriego/valet-linux v2.1.5
    - Conclusion: don't install cpriego/valet-linux v2.1.4
    - Conclusion: don't install cpriego/valet-linux v2.1.3
    - Conclusion: don't install cpriego/valet-linux v2.1.2
    - Conclusion: don't install cpriego/valet-linux v2.1.1
    - Conclusion: remove illuminate/container v6.3.0
    - Installation request for cpriego/valet-linux ^2.1 -> satisfiable by cpriego/valet-linux[v2.1.0, v2.1.1, v2.1.10, v2.1.2, v2.1.3, v2.1.4, v2.1.5, v2.1.6, v2.1.7, v2.1.8, v2.1.9].
    - Conclusion: don't install illuminate/container v6.3.0
    - cpriego/valet-linux v2.1.0 requires illuminate/container ~5.3 -> satisfiable by illuminate/container[5.7.17, 5.7.18, 5.7.19, v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9, v5.5.0, v5.5.16, v5.5.17, v5.5.2, v5.5.28, v5.5.33, v5.5.34, v5.5.35, v5.5.36, v5.5.37, v5.5.39, v5.5.40, v5.5.41, v5.5.43, v5.5.44, v5.6.0, v5.6.1, v5.6.10, v5.6.11, v5.6.12, v5.6.13, v5.6.14, v5.6.15, v5.6.16, v5.6.17, v5.6.19, v5.6.2, v5.6.20, v5.6.21, v5.6.22, v5.6.23, v5.6.24, v5.6.25, v5.6.26, v5.6.27, v5.6.28, v5.6.29, v5.6.3, v5.6.30, v5.6.31, v5.6.32, v5.6.33, v5.6.34, v5.6.35, v5.6.36, v5.6.37, v5.6.38, v5.6.39, v5.6.4, v5.6.5, v5.6.6, v5.6.7, v5.6.8, v5.6.9, v5.7.0, v5.7.1, v5.7.10, v5.7.11, v5.7.15, v5.7.2, v5.7.20, v5.7.21, v5.7.22, v5.7.23, v5.7.26, v5.7.27, v5.7.28, v5.7.3, v5.7.4, v5.7.5, v5.7.6, v5.7.7, v5.7.8, v5.7.9, v5.8.0, v5.8.11, v5.8.12, v5.8.14, v5.8.15, v5.8.17, v5.8.18, v5.8.19, v5.8.2, v5.8.20, v5.8.22, v5.8.24, v5.8.27, v5.8.28, v5.8.29, v5.8.3, v5.8.30, v5.8.31, v5.8.32, v5.8.33, v5.8.34, v5.8.35, v5.8.4, v5.8.8, v5.8.9].
    - Can only install one of: illuminate/container[5.7.17, v6.3.0].
    - Can only install one of: illuminate/container[5.7.18, v6.3.0].
    - Can only install one of: illuminate/container[5.7.19, v6.3.0].
    - Can only install one of: illuminate/container[v5.3.0, v6.3.0].
    - Can only install one of: illuminate/container[v5.3.16, v6.3.0].
    - Can only install one of: illuminate/container[v5.3.23, v6.3.0].
    - Can only install one of: illuminate/container[v5.3.4, v6.3.0].
    - Can only install one of: illuminate/container[v5.4.0, v6.3.0].
    - Can only install one of: illuminate/container[v5.4.13, v6.3.0].
    - Can only install one of: illuminate/container[v5.4.17, v6.3.0].
    - Can only install one of: illuminate/container[v5.4.19, v6.3.0].
    - Can only install one of: illuminate/container[v5.4.27, v6.3.0].
    - Can only install one of: illuminate/container[v5.4.36, v6.3.0].
    - Can only install one of: illuminate/container[v5.4.9, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.0, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.16, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.17, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.2, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.28, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.33, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.34, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.35, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.36, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.37, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.39, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.40, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.41, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.43, v6.3.0].
    - Can only install one of: illuminate/container[v5.5.44, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.0, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.1, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.10, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.11, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.12, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.13, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.14, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.15, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.16, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.17, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.19, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.2, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.20, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.21, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.22, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.23, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.24, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.25, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.26, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.27, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.28, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.29, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.3, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.30, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.31, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.32, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.33, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.34, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.35, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.36, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.37, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.38, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.39, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.4, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.5, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.6, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.7, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.8, v6.3.0].
    - Can only install one of: illuminate/container[v5.6.9, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.0, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.1, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.10, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.11, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.15, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.2, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.20, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.21, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.22, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.23, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.26, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.27, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.28, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.3, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.4, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.5, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.6, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.7, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.8, v6.3.0].
    - Can only install one of: illuminate/container[v5.7.9, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.0, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.11, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.12, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.14, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.15, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.17, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.18, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.19, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.2, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.20, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.22, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.24, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.27, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.28, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.29, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.3, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.30, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.31, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.32, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.33, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.34, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.35, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.4, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.8, v6.3.0].
    - Can only install one of: illuminate/container[v5.8.9, v6.3.0].
    - Installation request for illuminate/container (locked at v6.3.0) -> satisfiable by illuminate/container[v6.3.0].

**Comment by jmarcher** [comment]
@daniyum21 Please open a new issue, your problem does not seem to be related to this issue.

**Comment by Sunwarul** [comment]
Same problem here. Same message (PHP7.4 WSL2)

**Comment by arnab682** [comment]
How can I solve this...???

arnab@arnab-Inspiron-3521:~$ composer global require genesisweb/valet-linux-plus
Changed current directory to /home/arnab/.config/composer
Using version ^1.5 for genesisweb/valet-linux-plus
./composer.json has been updated
Running composer update genesisweb/valet-linux-plus
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - genesisweb/valet-linux-plus[1.5, ..., 1.5.3] require illuminate/container ~5.3 -> found illuminate/container[v5.3.0, ..., v5.8.36] but the package is fixed to v8.63.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires genesisweb/valet-linux-plus ^1.5 -> satisfiable by genesisweb/valet-linux-plus[1.5, 1.5.1, 1.5.3].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
